run-build-functions.sh

```diff
@@ -184,6 +187,14 @@ run_npm() {
     fi
   fi
 
+  if [ -n "$EXPERIMENTAL_NPM_WORKSPACES_CACHING" ]
+  then
+    restore_node_modules "npm"
+  else
+    restore_cwd_cache node_modules "node modules"
+  fi
+
+
```

Comment:
[sand] Small note: this file currently does not use double empty lines, so we might want to remove this empty line.

Reply:
Addressed in 68778bb9f90782fdfcdb4ac953bd6ff835bc8b30!